packages/teams-js/src/internal/communication.ts

```diff
@@ -63,7 +63,9 @@ interface InitializeResponse {
 export function initializeCommunication(
   validMessageOrigins: string[] | undefined,
   apiVersionTag: string,
+  customMessageIdStarter?: number,
 ): Promise<InitializeResponse> {
+  CommunicationPrivate.nextMessageId = customMessageIdStarter ?? CommunicationPrivate.nextMessageId;
```

Comment:
just a note: When we `ununinitializeCommunication`, we are still setting the nextMessageId back to zero instead of `customMessageIdStarter`

Reply:
I think we want to set it back to `customMessageIdStarter` in `uninitializeCommunication` too. @TrevorJoelHarris thoughts?